bluebrain/documentation/installing_with_environments.md

```diff
@@ -15,7 +15,7 @@ and then activating it, and fixing the configuration to consider all
 software together:
 
     $ spack env activate -d spamhameggs
-    $ spack config add config:spack:concretization=together
+    $ spack config add concretizer:unify:true
```

Comment:
I would put this after `create`. After `activate` looks like you need it every time you activate the env

Reply:
If you want the setting to apply to your environment, you need to activate it first. If you don't activate it first, the setting will go to `~/.spack/concretizer.yaml`, only to be overridden by your environment config.

Reply:
oh, that makes sense!

Reply:
Maybe 1 sentence to specify this. It was not clear to me

Reply:
OK, will add! Thanks for the feedback!